### PR TITLE
Create and use stylised checkboxes

### PR DESCRIPTION
**Pull request by JorikSchellekens**
They look like this now
image

### CODE REVIEW

File: src/components/views/elements/StyledCheckbox.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+interface IState {
+}
+
+export default class StyledCheckbox extends React.PureComponent<IProps, IState> {
+
+    public static readonly defaultProps = {
+        className: "",
+    }
+
+    constructor(props: IProps) {
```
Comment (t3chguy) [inline src/components/views/elements/StyledCheckbox.tsx:34]: whats the point of an empty constructor?

File: src/components/views/elements/StyledCheckbox.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        super(props);
+    }
+
+    public render() {
+        const { children, className, ...otherProps } = this.props;
+        // 56^10 so unlikely chance of collision.
+        const id = "checkbox_" + randomString(10);
```
Comment (t3chguy) [inline src/components/views/elements/StyledCheckbox.tsx:41]: changing ID on every render seems awful for AT (accessible technologies) which will have to rebuild their internal trees

Comment (JorikSchellekens) [reply src/components/views/elements/StyledCheckbox.tsx:41]: Fair, I'll add it to the constructor. (Knew, there was a reason I left it empty :P)

**Comment by JorikSchellekens** [comment]
Tests keep failing because it's matching against a weird branch.

**Comment by t3chguy** [comment]
just looks like your develop is out of date, pull in latest

**Pull request merged**